X86/X86MachineInstructionRaiserUtils.cpp

```diff
@@ -1867,6 +1867,20 @@ X86MachineInstructionRaiser::getGlobalVariableValueAt(const MachineInstr &MI,
             GlobalInit = ConstantInt::get(GlobalValTy, SV);
         }
 
+        std::string SymbolName = GlobalDataSymNameIndexStrRef.str();
+        // If there is a suffix like stdout@@GLIBC_2.2.5, remove it to check
+        // if the symbol is defined in a user-passed header file
+        if (SymbolName.find("@@") != std::string::npos) {
+          SymbolName = SymbolName.substr(0, SymbolName.find("@@"));
```

Comment:
Avoid calling `find` twice for the same `SymbolName`.

Reply:
Thanks!